githubclient.py

```diff
@@ -0,0 +1,20 @@
+from github import Github, GithubException
+
+
+def get_commits_github(reponame, since_date):
```

Comment:
Can you create new class ``GithubClient`` with getCommits(reponame, since_date) method? githubclient module can contain more methods in future and it is easier to replace object with another object with the same interface.

There are already methods that can go here: [1], [2]

[1] https://github.com/gofed/lib/blob/master/repositoryinfo.py#L210
[2] https://github.com/gofed/lib/blob/master/repositoryinfo.py#L221